### PR TITLE
clarify ipv4 address label on network creation

### PR DESCRIPTION
**Pull request by edlerd**
## Done

- clarify ipv4 address label on network creation.
- additional help text for v4/v6 address on the network form

### CODE REVIEW

File: src/pages/networks/forms/NetworkFormMain.tsx [hunk 1]
```diff
@@ -109,11 +110,12 @@ const NetworkFormMain: FC<Props> = ({ formik, project, isClustered }) => {
             getConfigurationRow({
               formik,
               name: "ipv6_address",
-              label: "IPv6 address",
+              label: "IPv6 address range",
               defaultValue: "auto",
               children: (
                 <IpAddressSelector
                   id="ipv6_address"
+                  family="IPv4"
```
Comment (Kxiru) [inline src/pages/networks/forms/NetworkFormMain.tsx:118]: Shouldn't this be IPv6? :)

**Review by Kxiru — CHANGES_REQUESTED** [review]
Small copy error, but other than this, this is good to go! :+1:

File: src/pages/networks/forms/NetworkFormMain.tsx (hunk 1 above)
Comment (edlerd) [reply src/pages/networks/forms/NetworkFormMain.tsx]: yes, good catch!

**Review by Kxiru — APPROVED** [review]
Looks good to me! :)

**Pull request merged**